src/Gemini.Modules.Inspector/Inspectors/NoUndoEditorBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Gemini.Modules.Inspector.Inspectors
+{
+    /// <summary>
+    /// This class is used as an editor base for values that do not support
+    /// undo / redo operations, for example write only variables.
+    /// </summary>
+    /// <typeparam name="TValue">Type of the value</typeparam>
+    public abstract class NoUndoEditorBase<TValue> : EditorBase<TValue>
+    {
+        public override TValue Value
+        {
+            get { return (TValue) BoundPropertyDescriptor.Value; }
```

Comment:
Yes, in my case the editor does not actually read the value, but I couldn't make it work with the default editor because the undo redo manager would read the value when setting it. I think this is useful for more cases than just write only values.